src/vos/evt_iter.c

```diff
@@ -154,16 +172,15 @@ evt_iter_probe_find(struct evt_iterator *iter, const struct evt_rect *rect)
 		return mid;
 
 	/* We didn't find the entry.  Find next entry instead */
+	/* NB: it does not need adjust for reverse iteration, because
+	 * mid <= anchor after the above while.
+	 */
```

Comment:
This looks incorrect to me, when reverse iterator failed to find anchor, it should the next smaller one. Could you revert this change?

Reply:
hmm, I think we do not need adjust the mid for reverse find, since the mid is already on the smaller side if it can not find.  Though we probably need check for start == end case.